DOCS/man/context_menu.rst

```diff
@@ -1,10 +1,49 @@
-CONTEXT MENU SCRIPT
-===================
-
-This script provides a context menu for platforms without integration with a
-native context menu. On these platforms, it can be disabled entirely using the
+CONTEXT MENU
+============
+
+The context menu is a menu that pops up on the video window. By default, it is
+bound to right click.
+
+menu.conf
+---------
+
+The default menu is defined in ``etc/menu.conf``. You can define your own menu
+in ``~~/menu.conf`` (see `FILES`_), or an alternative path specified with
+``--script-opt=select_menu_conf_path``. It is recommended to use the default
+``menu.conf`` from https://github.com/mpv-player/mpv/blob/master/etc/input.conf
+as an example to get started.
+
+Each line of ``menu.conf`` is a menu entry with fields separated by 1 or more
+tabs. The first field is the text shown in the menu. The second field is usually
+the command that is run when that entry is selected. Fields from the third
+onwards can specify ``checked=``, ``disabled=`` and ``hidden=`` states in the
+same way as `Conditional auto profiles`_.
+
+When there is only one field, or the second field specifies a state, it is
+interpreted as the title of an entry associated with a submenu. Fields below
+indented with leading whitespace are added to this submenu. Nested submenu items
+are defined by adding more leading whitespace than the parent menu entry.
+
+Empty lines are interpreted as separators.
+
+The second field can also be one of the following tokens to make that entry a
+submenu with the relative items: ``$playlist``, ``$video-tracks``,
+``$audio-tracks``, ``$sub-tracks``, ``$secondary-sub-tracks``, ``$chapters``,
+``$editions``, ``$audio-devices``. These menus are automatically disabled when
+empty.
```

Comment:
Those special things, like tracks are not working, they are not populated.

Reply:
Are you using `mpv-menu-plugin`? I changed `@` to `$` so they look more like variables though it is not important

Reply:
That was an unrelated error in the window scale conditions. Fixed that. Still no idea about the builtin submenus.